src/pages/AdminDashboard/UserDetails.tsx

```diff
@@ -0,0 +1,162 @@
+        },
+        list: {
+            paddingTop: 0,
+            paddingBottom: 0,
+        },
+        listItem: {
+            paddingTop: 0,
+            paddingBottom: 0,
+        },
+        apiTokenListItem: {
+            wordWrap: 'break-word',
+        },
+    })
+)
+
+const UserDetails = ({ onRoomEdit }) => {
+    const classes = useStyles()
+    const { rooms } = useSelector(selectRooms)
+    const { token, details } = useSelector(selectUser)
+    const { t } = useTranslation('dashboard')
+
+    const getSMSUsage = () => details?.usage?.sentSMSs
+
+    const getSMSQuota = () => details?.subscription?.quotas?.sms
+
+    return (
+        <>
+            <Paper elevation={0}>
+                <Grid container alignItems="center" spacing={2}>
+                    <Grid item>
+                        <ChatBubbleOutlineIcon className={classes.icon} />
+                    </Grid>
+                    <Grid item>
+                        <Typography variant="h6">
+                            {t('planned-discussions')}
+                        </Typography>
+                    </Grid>
+                </Grid>
+                <Divider />
+                <List component="nav" className={classes.list}>
+                    {rooms.map((room, i) => {
+                        return (
+                            <>
+                                <ListItem key={i} className={classes.listItem}>
+                                    <ListItemText primary={room.name} />
+                                    <ListItemText
+                                        primary={formatStartAtDate(room)}
+                                        primaryTypographyProps={{
+                                            align: 'right',
+                                        }}
+                                    />
+                                    <IconButton
+                                        onClick={() => onRoomEdit(room)}>
+                                        <EditIcon />
+                                    </IconButton>
+                                </ListItem>
+                                <Divider variant="middle" />
+                            </>
+                        )
+                    })}
+                </List>
+            </Paper>
+            <Box m={6} />
+            <Paper elevation={0}>
+                <Grid container alignItems="center" spacing={2}>
+                    <Grid item>
+                        <RestaurantIcon className={classes.icon} />
+                    </Grid>
+                    <Grid item>
+                        <Typography variant="h6">{t('consumption')}</Typography>
+                    </Grid>
+                </Grid>
+                <Divider />
+                <ListItem className={classes.list}>
+                    {details && (
+                        <ListItemText
+                            primary={`${
+                                getSMSUsage() !== undefined
```

Comment:
for better readability
`primary={getSMSUsage() || '?'}`

Reply:
I also replaced the first '?' with '0'